### ISSUE TITLE
Loading BERT checkpoints into TLM from TensorFlow

### ISSUE THREAD

**Issue by dpressel** [dpressel/mead-baseline#528]
We have a solution for loading BERT checkpoints into PyTorch 8 mile layers directly from huggingface checkpoints by modifying the keys in the pytorch state dict.  What would be better is to use this code to then re-write the checkpoint in TLM NPZ format so it can be loaded into TensorFlow 8 mile layers as well.  The NPZ files can be hosted on dropbox and has the advantage that the same checkpoint can be used in pytorch and TF

**Comment by dpressel** [comment]
This PR makes TF support the same options as PyTorch including layer norm epsilon and T2T-style LN (after) https://github.com/dpressel/mead-baseline/pull/537

**Comment by dpressel** [comment]
Waiting on checkpoint generation for verification
